recipe/meta.yaml

```diff
@@ -22,14 +24,53 @@ requirements:
   host:
     - ncurses
 
-test:
-  commands:
-    - test -f $PREFIX/lib/libestbase.so  # [linux]
-    - test -f $PREFIX/lib/libestbase.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libestools.so  # [linux]
-    - test -f $PREFIX/lib/libestools.so.2.5  # [linux]
-    - test -f $PREFIX/lib/libeststring.so  # [linux]
-    - test -f $PREFIX/lib/libeststring.so.2.5  # [linux]
+outputs:
+  - name: libestools
+    script: install.sh  # [linux]
+    test:
+      commands:
+        - test -f $PREFIX/lib/libestbase.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libestbase.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libestbase.so.2.5  # [linux]
+        - test -f $PREFIX/lib/libestools.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libestools.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libestools.so.2.5  # [linux]
+        - test -f $PREFIX/lib/libeststring.so.{{ version }}  # [linux]
+        - test -L $PREFIX/lib/libeststring.so.2.5  # [linux]
+        - test -e $PREFIX/lib/libeststring.so.2.5  # [linux]
+  - name: libestools-dev
+    script: install_devel.sh  # [linux]
+    requirements:
+      host:
+        - {{ pin_subpackage("libestools", exact=True) }}
+      run:
+        - {{ pin_subpackage("libestools", exact=True) }}
+    test:
+      commands:
+        - test -f $PREFIX/include/speech_tools/EST.h  # [linux]
+        - test -f $PREFIX/lib/speech_tools/config/config  # [linux]
+        - test -f $PREFIX/lib/speech_tools/lib/siod/init.scm  # [linux]
+        - test -L $PREFIX/lib/libestbase.so  # [linux]
+        - test -e $PREFIX/lib/libestbase.so  # [linux]
+        - test -L $PREFIX/lib/libestools.so  # [linux]
+        - test -e $PREFIX/lib/libestools.so  # [linux]
+        - test -L $PREFIX/lib/libeststring.so  # [linux]
+        - test -e $PREFIX/lib/libeststring.so  # [linux]
+  - name: speech-tools
+    script: install_bin.sh  # [linux]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - libgomp  # [linux]
+      host:
+        - libestools
+      run:
+        - libestools
+    test:
+      commands:
+        - test -f $PREFIX/bin/na_play  # [linux]
+        - test -f $PREFIX/bin/siod  # [linux]
```

Comment:
Use a jinja for loop to check for every binary.

https://github.com/conda-forge/arrow-cpp-feedstock/blob/3339a3f8043d46b57fd9cecd47c18ef0b7add8ba/recipe/meta.yaml#L157

Reply:
What I meant was that you are installing 20+ binaries, but you are only checking for two of them. Using a jinja loop makes it easier to check all of them:

```yaml
        {% set files = [
          "na_play", "siod", .... "wf_strun"
        ] %}
        {% for f in files %}
        - test -f $PREFIX/bin/{{ f }}  # [linux]
        {% endfor %}
```
without having to re-type `test -f $PREFIX/bin/` 20+ times.